### PR TITLE
started setting up user company join in redux.

### PR DESCRIPTION
**Pull request by IsmaelPerezGit**
Haven't been able to make the company form functional. So far, I've set up the redux portion but am not completely sure I'm taking the right approach. Lets discuss it when theres a chance.

### CODE REVIEW

**Comment by chrisjannenga** [comment]
Sounds good to me!

File: src/Components/Forms/Company.js [hunk 1]
```diff
@@ -33,12 +35,30 @@ class Company extends Component {
         );
     }
 
+    onButtonPress() {
+        const {name, styleGuide, primaryGoal} = this.state;
+        // if (name === '' || styleGuide === '' || primaryGoal === '') {
+        //     return alert('Must fill in all fields')
+        // }
+        return (() => {
+            this.props.updateCompanyInfo({
```
Comment (g-dylan-cunningham) [inline src/Components/Forms/Company.js]: Does this work? If not, I've just been calling the dispatch function very simply...
```
onButtonPress(){
     this.props.updateCompanyInfo(obj)
}
```
and it works. Hope I didn't throw you off with all of that talk about making anonymous functions after .then...

Comment (IsmaelPerezGit) [reply src/Components/Forms/Company.js]: no worries. The anonymous function was actually one of the few things that made sense to me haha. The post at least when i was testing wasnt working for me. I thought maybe it had to do with not having the company_id that is generated through the back end with the random number function you wrote.

**Comment by g-dylan-cunningham** [comment]
Not sure about the route. It seemed to be working before - that userCompanyJoin is pretty tightly coupled with company and user. Maybe the params or body aren't quite right? The number generator seems to work. https://repl.it/@flacallama/AngelicGroundedCurrencies

**Pull request merged**